### PR TITLE
benmalka/foxdriver to saucelabs/foxdriver

### PR DESCRIPTION
**Pull request by benmalka**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Firefox Remote Debugging Protocol consists of multiple actors that provide d
 __Example__ - opening page and get console.logs
 
 ```js
-import Foxdriver from 'foxdriver'
+import Foxdriver from '@benmalka/foxdriver'
```
Comment (christian-bromann) [inline docs/README.md:21]:
```suggestion
import Foxdriver from 'foxdriver'
```

File: docs/README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ You can also attach yourself to an already running Firefox browser. This require
 To attach yourself to the browser you then need to create a Foxdriver instance with the correct port and host and call the `connect()` method:
 
 ```js
-import Foxdriver from 'foxdriver'
+import Foxdriver from '@benmalka/foxdriver'
```
Comment (christian-bromann) [inline docs/README.md:50]:
```suggestion
import Foxdriver from 'foxdriver'
```

File: lib/launcher.js [hunk 3]
```diff
@@ -65,11 +99,8 @@ export default class Launcher {
                 windowsFirefoxDirectory = path.join(prefix, suffix)
                 fs.accessSync(windowsFirefoxDirectory)
                 return windowsFirefoxDirectory
-            } catch (e) {
-                return windowsFirefoxDirectory
-            }
+            } catch (e) {}
```
Comment (christian-bromann) [inline lib/launcher.js:102]: Should we at least log the error?

File: package.json [hunk 4]
```diff
@@ -4,10 +4,10 @@
   "author": "Christian Bromann <christian@saucelabs.com>",
   "description": "Foxdriver is a Node library which provides a high-level API to control Firefox over the Remote Debugging Protocol",
   "license": "Apache-2.0",
-  "homepage": "https://github.com/saucelabs/foxdriver#readme",
+  "homepage": "https://github.com/benmalka/foxdriver#readme",
```
Comment (christian-bromann) [inline package.json:7]:
```suggestion
  "homepage": "https://github.com/saucelabs/foxdriver#readme",
```

File: package.json [hunk 5]
```diff
@@ -4,10 +4,10 @@
...
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/saucelabs/foxdriver.git"
+    "url": "git+https://github.com/benmalka/foxdriver.git"
```
Comment (christian-bromann) [inline package.json:10]:
```suggestion
    "url": "git+https://github.com/saucelabs/foxdriver.git"
```

File: package.json [hunk 6]
```diff
@@ -19,14 +19,14 @@
     "devtools"
   ],
   "bugs": {
-    "url": "https://github.com/saucelabs/foxdriver/issues"
+    "url": "https://github.com/benmalka/foxdriver/issues"
```
Comment (christian-bromann) [inline package.json:22]:
```suggestion
    "url": "https://github.com/saucelabs/foxdriver/issues"
```

**Review by christian-bromann — COMMENTED** [review]
This is great 👏 , couple of comments

**Review by christian-bromann — APPROVED** [review]
This LGTM 👍  thank you very much!

@farhan-sauce can you take a second look

**Comment by farhan-sauce** [comment]
Thank you @benmalka 
It looks like https://github.com/saucelabs/firefox-adapter doesn't work as expected with these changes. I will debug more on this.

**Comment by christian-bromann** [comment]
IMO this should not be a blocker.

**Comment by benmalka** [comment]
> Thank you @benmalka
> It looks like https://github.com/saucelabs/firefox-adapter doesn't work as expected with these changes. I will debug more on this.

Can you shed some light on this? Maybe I can help you pinpoint the issue

**Review by farhan-sauce — APPROVED** [review]
LGTM 👍

**Comment by christian-bromann** [comment]
Thank you very much @benmalka 🙏

**Pull request merged**